### ISSUE TITLE
MindTheGapps fails to download google play updates

### ISSUE THREAD

**Issue by jeremyfix** [fakemanoan/LineageOS-Releases#101]
### Build used

lineage-20.0-20240924-UNOFFICIAL-zerofltexx.zip

### Model number

Galaxy A6 flat

### Describe the bug

Flashing Lineage OS + MindTheGapps worked and I successfully booted into the OS;

However, during the initial setup, after configuring Family link (I do not know if this is relevant or not.. I though possibly), google wants to download updates for google play and this is stalling; I do not see any progress;

I checked if this might be related to access to internet but however I was :

1) able to navigate on a website
2) the google account was correctly recognized 

I'm currently giving a try to NikGapps.

### To Reproduce

1. flash lineage OS + the provided MindTheGapps for Android 13
2. Restart and perform the initial setup

**Comment by A-Australian** [comment]
Yeah it didn't work for me ethier, I recommend nikgapps (basic) as that's what worked for me.

**Comment by fakemanoan** [comment]
please install gapps prior to first boot
